### ISSUE TITLE
Feature Request: Booked out weight

### ISSUE THREAD

**Issue by TechJed** [CompositeLight/currentRMS-helper#57]
Would be great when loading multiple vans if there was a total booked out weight, similar to how the containers work

**Comment by CompositeLight** [comment]
Good idea - I can see how this would be useful. Should be possible to bolt this on to the new container weights code - will get to it when I can.

**Comment by TechJed** [comment]
This works great but seems to have a lot more decimal places than current can store. Screenshot attached 
![image_2024-08-16_103018071](https://github.com/user-attachments/assets/e25f8dc8-074c-4ee4-a407-f230306fe0b1)

**Comment by CompositeLight** [comment]
I think I know what I did wrong here. Have just pushed a fix. Let me know if that sorts it.

**Comment by TechJed** [comment]
Still got the same issue
![image_2024-09-11_124827879](https://github.com/user-attachments/assets/9bb1eb8a-ef9c-4052-bb14-65033ce21573)

**Comment by CompositeLight** [comment]
Very weird - this should be impossible, as the weight data is coerced into a number every time. I can't replicate it at all. Could you give me more info on how this presents: Does it happen on every job, if not what is unique about the ones with the error? Does it happen immediately (from the first book out scan), or do the extra digits row over time? Any other clues welcome.

**Comment by CompositeLight** [comment]
FINALLY cracked the crazy decimal thing. Bug fix coming in v.1.31.19.

**Comment by TechJed** [comment]
Woooo
